src/commands/edit-input-schema.js

```diff
@@ -0,0 +1,120 @@
+const fs = require('fs');
+
+const detectIndent = require('detect-indent');
+const express = require('express');
+const open = require('open');
+
+const { ApifyCommand } = require('../lib/apify_command');
+const outputs = require('../lib/outputs');
+
+const DEFAULT_INPUT_SCHEMA_PATH = './INPUT_SCHEMA.json';
+const INPUT_SCHEMA_EDITOR_BASE_URL = `https://apify.github.io/input-schema-editor-react/`;
+const INPUT_SCHEMA_EDITOR_ORIGIN = new URL(INPUT_SCHEMA_EDITOR_BASE_URL).origin;
+
+class EditInputSchemaCommand extends ApifyCommand {
+    async run() {
+        const { args } = this.parse(EditInputSchemaCommand);
+        const { path = DEFAULT_INPUT_SCHEMA_PATH } = args;
+
+        outputs.warning('This command is still experimental and might break at any time. Use at your own risk.\n');
+        outputs.info(`Editing input schema at "${path}"...`);
+
+        let server;
+        let jsonIndentation = '    ';
+        let appendFinalNewline = true;
+
+        const app = express();
+        app.use(express.json());
+        app.use((req, res, next) => {
+            res.header('Access-Control-Allow-Origin', INPUT_SCHEMA_EDITOR_ORIGIN);
+            res.header('Access-Control-Allow-Headers', 'Content-Type');
+            res.set('Connection', 'close');
+            next();
+        });
+
+        app.get('/api/input-schema', (req, res) => {
```

Comment:
'Even though it's experimental, I'd prefix this with version `v1/` to make sure we can eventually do changes later on.

Reply:
Or maybe later, we can use dates versioning in the header, and this would simply become the default... . Date versioning might be better for an internal endpoint.

Reply:
Good point, I will add the date-based API versioning, it should not be hard. It will be useful if this proves buggy and we have to do some breaking changes.